Umbraco Commerce Taxes Documentation (#6380)
umbraco/UmbracoDocs: 14/umbraco-commerce/reference/taxes/calculated-tax-rates.md

﻿---
description: Calculated Rate Taxes in Umbraco Commerce.
---

# Calculated Tax Rates

> Fixed rate taxes allows you to dynamically calculate the tax obligations of an order by using third party calculation platforms. This is a more complex option that is useful for countries that have different tax rates for different types of products or for different regions within the country (the norm in the US).

Reviewer: ```suggestion
Fixed-rate taxes allow you to dynamically calculate the tax obligations of an order by using third-party calculation platforms. This complex option is useful for countries with different tax rates for different product types or regions within the country (the norm in the US).
```
